apps/web/src/screens/Dashboard/RemoveLayout/RemoveSymbol.tsx

```diff
@@ -0,0 +1,30 @@
+const RemoveSymbol = () => {
+	return (
+		<svg
```

Comment:
use `Svg`, and other component from 'react-native-svg'; this won't work on Mobile